docs/sample_schema_package/README

```diff
@@ -1,3 +1,6 @@
+SPDX-FileCopyrightText: 2024 Red Hat, Inc
+SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
This should be commented out but it's a plain text file. Maybe rename this file to `.md` and add the tags in a comment, as for `SECURITY.md`.